docs/en/index.md

```diff
@@ -0,0 +1,341 @@
+---
+title: Silverstripe elemental
+---
+
+# Silverstripe elemental
+
+## Introduction
+
+This module extends a page type to swap the content area for a list of manageable elements to compose a page out
+of rather than a single text field. Features supported:
+
+- Versioning of elements
+- Ability to add, remove supported elements per *elemental area*
+
+The module provides basic markup for each of the elements but you will likely need to provide your own styles. Replace
+the `$Content` variable with `$ElementalArea` in your page templates, and rely on the markup of the individual elements.
+
+For a more detailed overview of using this module, please see [the User help guides](docs/en/userguide/index.md).
+
+## Requirements
+
+- Silverstripe CMS ^5.0
+- Versioned Admin ^2.0
+- GridFieldExtensions ^4.0
+
+For a CMS 4.3+ compatible version of this module, please see the [4.x release line](https://github.com/silverstripe/silverstripe-elemental/tree/4#readme).
```

Comment:
```suggestion
```
Requirements are in composer.json, no need to state them here - nor to mention CMS 4 in the CMS 5 docs.

Reply:
Updated